src/main/java/org/llorllale/cactoos/matchers/InputHasContent.java

```diff
@@ -29,27 +29,21 @@
 import org.cactoos.Input;
 import org.cactoos.Text;
 import org.cactoos.text.TextOf;
-import org.cactoos.text.UncheckedText;
-import org.hamcrest.Description;
 import org.hamcrest.Matcher;
-import org.hamcrest.TypeSafeMatcher;
 
 /**
  * Matcher for the input.
  *
  * @since 0.11
+ * @checkstyle IndentationCheck (500 lines)
```

Comment:
@Iprogrammerr why? I believe you should be able to fix the indentation instead of ignoring the problem

Reply:
![lambda](https://user-images.githubusercontent.com/41533396/51272711-24ebcf00-19cb-11e9-81e3-9fdc40da9d1a.JPG)
I do not know what is going out there. I tried every possible identation, new lines configuration etc. Qulice is not happy with any of them.

Reply:
@Iprogrammerr you should open an issue in qulice :) but I'm good with that then, let's see if @llorllale is ok too :)

Reply:
@victornoel Here it [is](https://github.com/teamed/qulice/issues/985).

Reply:
@Iprogrammerr I'd prefer if you fix the issue. If you can't, you can deactivate this checkstyle rule on the lines specific to the problem, open a ticket to qulice and leave a todo for this.